app/src/main/java/com/aguesoftguar/medalarm/data/source/PatientsDataSource.java

```diff
@@ -18,6 +18,7 @@
 
 import android.support.annotation.NonNull;
```

Comment:
There are indentation failures in this class.

Please, execute `Code\Reformat code` in AS